### DISCUSSION TITLE
hide statusline unless invoked

### DISCUSSION THREAD

**Discussion by gennaro-tedesco** [saulpw/visidata#2150] [Q&A] [answered]
Could you please address me to the relevant documentation (if existing, I may have overlooked it) on whether it is possible to hide the bottom statusline and only invoke it on demand (say when entering a command or a search)?

Very often I use `visidata` to pipe the output of simple commands in, namely as some sort of quick pager, and I would prefer to have more room available to display some more rows of data in the bottom at the cost of the statusline (which for those use cases I would not use).

**Comment by saulpw** [comment, marked as answer]
Hi @gennaro-tedesco, the status line can't be disabled.  This seemed like a reasonable requirement, as even the classic pagers `less` and `more` use the bottom line for context information.

**Reply by gennaro-tedesco** [reply]
I see, good so then and thank you for answering :)
